src/Rules/Properties/OverridingPropertyRule.php

```diff
@@ -76,6 +76,14 @@ public function processNode(Node $node, Scope $scope): array
 					$prototype->getDeclaringClass()->getDisplayName(),
 					$node->getName(),
 				))->nonIgnorable()->build();
+			} else {
+				$errors[] = RuleErrorBuilder::message(sprintf(
```

Comment:
I was not able to come up with a similar example for the `static` case above this IF-block, therefore only added here.

Reply:
just read about https://github.com/phpstan/phpstan-src/commit/3e383fc9620a46173651402049a8e8c4904591b6 which resolves my questions, as readonly properties cannot be static.